packages/pangea-node-sdk/src/services/audit.ts

```diff
@@ -532,6 +532,18 @@ class AuditService extends BaseService {
       return root;
     };
 
+    const fixConsistencyProof = async (treeSize: number) => {
+      const response = await this.root(treeSize);
+      const root: Audit.Root = response.result.data;
```

Comment:
Could use `localRoot()`?
```suggestion
      const root = await localRoot(treeSize);
```